### PR TITLE
Add description column to epoches table and show_pending_gives to circles

### LINKED ISSUE #1572: Epoch Meta Data or Notes
- People want to be able to write info about what the epoch is about.
- This would help add flavor to the epoch

In this design, we remove epoch naming such as epoch 1, epoch 2, etc., just allowing the date range to be the name. The user can now add a description to upcoming or current epochs. 

[UI](https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/Coordinape-App?node-id=26295%3A76370)
[Prototype](https://www.figma.com/proto/L4oZwYVUdpgacZtwipGaYe/Coordinape-App?page-id=14369%3A30224&node-id=26295%3A75992&viewport=1239%2C18819%2C0.27&scaling=min-zoom&starting-point-node-id=14370%3A52369&show-proto-sidebar=1&hide-ui=1)

### PR DESCRIPTION
**Pull request by sshmm**
## Motivation and Context

partially resolves #1572 and #1523 - Backend part

## Description
- add description column to epoches table to hold the description of the epoch
- add show_pending_gives to circles table to hold show gives during epoch

### CODE REVIEW

**Comment by sshmm** [comment]
I will create one PR for backend and front end

File: hasura/migrations/default/1668026379094_alter_table_public_epoches_add_column_description/up.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alter table "public"."epoches" add column "description" varchar
```
Comment (zashton) [inline hasura/migrations/default/1668026379094_alter_table_public_epoches_add_column_description/up.sql:1]: why not use a `text` type ?

**Review by zashton — CHANGES_REQUESTED** [review]
thanks for the PR, looks good to me, just one nit on the description type.

File: hasura/migrations/default/1668026379094_alter_table_public_epoches_add_column_description/up.sql (hunk 1 above)
Comment (sshmm) [reply hasura/migrations/default/1668026379094_alter_table_public_epoches_add_column_description/up.sql:1]: @zashton  I set the maximum to 100 characters so far in the form. should I change it to text?

Comment (topocount) [reply hasura/migrations/default/1668026379094_alter_table_public_epoches_add_column_description/up.sql:1]: @sshmm there is no semantic difference under the hood between `varchar` and `text`. This is just for consistency.

File: hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+alter table "public"."circles" drop column "show_pending_gives";
```
Comment (topocount) [inline hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql:1]: glad to have a drop script!

nit: please at the trailing newline here

**Review by topocount — APPROVED** [review]
Looks good to me, pending the requested change from zashton

File: hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql (hunk 2 above)
Comment (sshmm) [reply hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql:1]: it gets deleted when I click save by prettier

Comment (sshmm) [reply hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql]: done by github editor

**Review by zashton — APPROVED** [review]

File: hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql (hunk 2 above)
Comment (topocount) [reply hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql]: > it gets deleted when I click save by prettier

will look into this

**Pull request merged**